local-notifications.android.js

```diff
@@ -1,7 +1,7 @@
 var application = require("application");
 var utils = require("utils/utils");
 var LocalNotifications = require("./local-notifications-common");
-
+var context = utils.ad.getApplicationContext();
```

Comment:
I'm not entirely sure this will always resolve to an object (fi. during app start).. I will test it though.. if not I'll move this to any place where `context` is actually use.